### PR TITLE
Missing submit button

### PR DESCRIPTION
**Pull request by raul-marquez-csa**
Issue: [[Bug] The submit button is not appearing #60](https://github.com/project-chip/certification-tool/issues/60)
Frontend PR: [Missing submit button #14](https://github.com/project-chip/certification-tool-frontend/pull/14)

Missing submit button fixed, see frontend PR description for testing results.

### CODE REVIEW

File: app/user_prompt_support/user_prompt_manager.py [hunk 1]
```diff
@@ -50,6 +50,15 @@ def __init__(self, prompt: PromptRequest, message_id: int) -> None:
...
+        # Check if options exist
+        if "options" in prompt_dict:
+            if prompt_dict["options"]:
+                message_type = MessageTypeEnum.OPTIONS_REQUEST
+            else:
+                message_type = MessageTypeEnum.MESSAGE_REQUEST
+                
         message_dict = {
             MessageKeysEnum.TYPE: MessageTypeEnum.PROMPT_REQUEST,
             MessageKeysEnum.PAYLOAD: prompt_dict,
```
Comment (antonio-amjr) [inline app/user_prompt_support/user_prompt_manager.py:64]: As the Flake check alerted, we need to use the `message_type` created:
```suggestion
        message_dict = {
            MessageKeysEnum.TYPE: message_type,
            MessageKeysEnum.PAYLOAD: prompt_dict,
```

**Review by antonio-amjr — CHANGES_REQUESTED** [review]
Minor change pointed by the GitHub checking:

File: app/constants/websockets_constants.py [hunk 2]
```diff
@@ -25,6 +25,8 @@
 # Enum Keys for different types of messages currently supported by the tool
 class MessageTypeEnum(str, Enum):
     PROMPT_REQUEST = "prompt_request"
+    OPTIONS_REQUEST = "options_request"
+    MESSAGE_REQUEST = "message_request"
```
Comment (hiltonlima) [inline app/constants/websockets_constants.py:29]:
```suggestion
    MESSAGE_REQUEST = "message_request"
```

**Review by hiltonlima — APPROVED** [review]

**Review by rquidute — APPROVED** [review]

**Review by antonio-amjr — APPROVED** [review]

File: app/user_prompt_support/user_prompt_manager.py [hunk 3]
```diff
@@ -50,8 +50,17 @@ def __init__(self, prompt: PromptRequest, message_id: int) -> None:
...
         prompt_dict[MESSAGE_ID_KEY] = self.message_id
+        message_type = MessageTypeEnum.PROMPT_REQUEST
+
+        # Check if options exist
+        if "options" in prompt_dict:
+            if prompt_dict["options"]:
+                message_type = MessageTypeEnum.OPTIONS_REQUEST
+            else:
+                message_type = MessageTypeEnum.MESSAGE_REQUEST
+
```
Comment (ccruzagralopes) [inline app/user_prompt_support/user_prompt_manager.py:61]:
```suggestion

```

**Pull request merged**